### DISCUSSION TITLE
Welcome to ko6a Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [moncke/ko6a#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about 🤔
  * Share ideas 💸
  * Engage with other community members 📯
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eric-hc** [comment]
Hi I'm Eric and I am trying to start this project!

**Comment by tiffkwin** [comment]
Hi I'm Tiff and I am helping Eric start this project!

**Comment by ghost** [comment]
yo
